amundsen_application/api/metadata/v0.py

```diff
@@ -54,14 +54,14 @@ def popular_tables() -> Response:
     """
     try:
         url = app.config['METADATASERVICE_BASE'] + POPULAR_TABLES_ENDPOINT
-        response = request_metadata(url=url)
+        data = {'limit': app.config['POPULAR_TABLE_COUNT']}
```

Comment:
Can your PR link to the metadata implementation as well, I'm not seeing where this gets used in the endpoint.

Additionally `GET` requests don't send request data, instead I would expect we would include the parameters in the url in query string format, and the metadata service will parse that value to use it accordingly.

Reply:
[This](https://github.com/lyft/amundsenmetadatalibrary/blob/master/metadata_service/api/swagger_doc/popular_tables_get.yml) is the swagger doc for popular tables metadata API, and thanks for pointing out the `GET` request stuff.

Reply:
I'm not seeing the approach of using `params` work locally for us. I'm not sure why, but I have reservations about how widely supported is `params` in `GET` requests are for all different implementations of request clients.

Existing code in this repository also uses query strings and we don't plan on changing that precedent, so I'd suggest:
```
service_base = app.config['METADATASERVICE_BASE']
count = app.config['POPULAR_TABLE_COUNT']
url = f'{service_base}{POPULAR_TABLES_ENDPOINT}?limit={count}'
```
which also removes the need for any changes in `request_utils.py`.